3-PB/Documentazione_esterna/Piano_di_qualifica/cruscotto.tex

```diff
@@ -31,7 +33,7 @@ \subsection{Analisi}
 Dopo un ottimo inizio nei primi due sprint, è possibile notare un calo dei lavori nel terzo sprint, e un altro leggero rallentamento nel quinto. L'Estimate to Complete al terzo e al quinto sprint è diminuito in modo meno marcato rispetto lo sprint precedente. Inoltre, l'Estimate at Completion ha superato il Budget at Completion.\\
 Questo è causato dal mancato raggiungimento di tutti gli obiettivi fissati dalla milestone del terzo sprint.\\
 Il valore dopo il terzo sprint, nonostante il brusco rallentamento dell'ottavo periodo di osservazione, è comunque rimasto stabile ad un valore poco al di sopra del Budget at Completion.
-
+Nei successivi periodi, l'ETC scende costantemente, fatta eccezione per l'ottavo sprint, in cui la diminuzione è maggiore, per le problematiche descritte precedentemente. In seguito, l'Estimate to Complete riprende un andamento discendente costante. 
```

Comment:
Nei successivi periodi, l'ETC scende costantemente, in particolare dall'ottavo sprint, in cui la diminuzione è maggiore, per l'aumento delle ore produttive e dell'impegno del gruppo.